docs/source/install-kube.rst

```diff
@@ -216,9 +216,9 @@ There are no other requirements for images, any image that meets the above
 should work fine. You may install any additional libraries or dependencies you
 require.
 
-To develop your own image, you may either base it on a compatible version of
-``daskgateway/dask-gateway``, or use our `example dockerfile`_ as a reference
-and develop your own.
+We encourage you to maintain your own image for scheduler and worker pods as
+this project only provides a `minimal image`_ for testing purposes which can be
+used as inspiration, but isn't recommended to be used as a base image.
```

Comment:
```suggestion
used as inspiration, but isn't recommended to be used in production.
```

Reply:
It can be used in production, and I wouldn't even say we don't recommend using it as a base image, it's just that we won't do anything extra to support using it as a base image (adding extra build args, etc...).